### ISSUE TITLE
Dashboard title should have arial-level attribute

### ISSUE THREAD

**Issue by iethree** [metabase/metabase#70544]
Dashboard title elements should have the aria-level attribute 

<img width="363" height="167" alt="Image" src="https://github.com/user-attachments/assets/32926e4b-8436-4b39-8cbc-a6be106afb4e" />
